my_game.py

```diff
@@ -352,11 +371,192 @@ def on_mouse_press(self, x: int, y: int, button: int, modifiers: int):
         """
         if arcade.get_distance(x, y, CONFIG['BUTTON_X'], CONFIG['BUTTON_Y']) < self.play_button.width // 2:
             self.start_game()
+            
+        if arcade.get_distance(x, y, CONFIG['SETTINGS_BUTTON_X'], CONFIG['SETTINGS_BUTTON_Y']) < self.settings_button.width // 2:
+            self.enter_settings()
 
     def start_game(self):
         in_game_view = InGameView()
         self.window.show_view(in_game_view)
 
+    def enter_settings(self):
+        settings_view = SettingsView()
+        self.window.show_view(settings_view)
+            
+            
+class SettingsView(arcade.View):
+    """
+    Veiw for the Settings Screen
+    """
+
+    # Making dict for keys and key IDs
+    keys = dir(arcade.key)
+    key_to_id = {}
+    id_to_key = {}
+    non_keys = 0
+    for a in keys:
+        if a[0] == "_":
+            non_keys += 1
+    for n in range(non_keys):
+        keys.pop(-1)
+    for k in keys:
+        key_to_id.update({k: eval("arcade.key." + k)})
+    # Remove the MOTION keys because they are just duplicates
+    key_to_id.pop("MOTION_LEFT")
+    key_to_id.pop("MOTION_RIGHT")
+    key_to_id.pop("MOTION_UP")
+    key_to_id.pop("MOTION_DOWN")
+    for k in key_to_id:
+        id_to_key.update({key_to_id[k]: k})
+
+    def __init__(self):
+        super().__init__()
+        
+        self.change_settings_guide = True
+        self.change_thrust_key = False
+        self.change_fire_key = False
+        self.change_turn_right_key = False
+        self.change_turn_left_key = False
```

Comment:
Can't we have a variable that says what needs to be changed instead of having all these variables? Something like:
```
key_to_change = "fire"
```

default value would be _None_

Reply:
That makes sense. I'll implement that.